src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyNotModifiedResponse.java

```diff
@@ -153,6 +164,16 @@ private void onAbort(AbortFW abort)
                                abort.trace());
     }
 
+    private void onReset(ResetFW reset)
+    {
+        //NOOP
+    }
+
+    private void onWindow(WindowFW window)
+    {
+        //NOOP
```

Comment:
This is a no-op instead of `doEnd` because we send `doEnd` in reaction to `END` instead?

Reply:
yep.